PySubtitle/Helpers/Localization.py

```
@@ -69,3 +76,70 @@ def tr(context: str, text: str) -> str:
     return _(text)
 
 
+def get_available_locales() -> List[str]:
+    """
+    Scan the locales directory and return a list of available locale codes.
+    """
+    locales_dir = _get_locale_dir()
+    languages = []
+    try:
+        for name in os.listdir(locales_dir):
+            path = os.path.join(locales_dir, name, 'LC_MESSAGES')
+            if os.path.isdir(path):
+                # Check if there's actually a .po or .mo file in this locale
```

Good instinct but unnecessary - assume that if the locale directory exists it will be populated at this stage. We can raise an error when the locale is selected if the translations are not found.

---

@copilot Good instinct but unnecessary - assume that if the locale directory exists it will be populated at this stage. We can raise an error when the locale is selected if the translations are not found.